### PR TITLE
DOC-6944 added clarity around backups post deletion

### PR DESCRIPTION
**Pull request by ashleykwtam**
Addresses DOC-6944:
- Currently our docs do not inform the user what happens to backups after a cluster is deleted. This PR adds that update
- Full backups are retained for 30 days after a cluster is deleted, incremental backups for 7
- If a user deletes the organization they will lose access to all of the backups that were taken from that cluster.

Preview:
- [Cloud FAQs Dedicated](https://deploy-preview-17193--cockroachdb-docs.netlify.app/docs/cockroachcloud/frequently-asked-questions#who-is-responsible-for-backup)
- [Cloud FAQs Serverless](https://deploy-preview-17193--cockroachdb-docs.netlify.app/docs/cockroachcloud/serverless-faqs#can-i-backup-my-cockroachdb-serverless-cluster-does-cockroach-labs-take-backups-of-my-cluster)

### CODE REVIEW

File: src/current/cockroachcloud/frequently-asked-questions.md [hunk 1]
```diff
@@ -95,20 +95,20 @@ We do not automatically scale nodes based on your capacity usage. To add or remo
...
 {{site.data.alerts.end}}
 
-The backups for AWS clusters are encrypted using [AWS S3’s server-side encryption](https://docs.aws.amazon.com/AmazonS3/latest/dev/UsingServerSideEncryption.html) and the backups for GCP clusters are encrypted using [Google-managed server-side encryption keys](https://cloud.google.com/storage/docs/encryption/default-keys).
+The backups for AWS clusters are encrypted using [AWS S3’s server-side encryption](https://docs.aws.amazon.com/AmazonS3/latest/dev/UsingServerSideEncryption.html) and the backups for GCP clusters are encrypted using [Google-managed server-side encryption keys](https://cloud.google.com/storage/docs/encryption/default-keys). You can also [back up and restore](take-and-restore-customer-owned-backups.html) data on your own. If you need additional help, [contact us](https://support.cockroachlabs.com).
```
Comment (MichaelTrestman) [inline src/current/cockroachcloud/frequently-asked-questions.md:106]: I suggest moving this bit up higher (see other suggestion above).
```suggestion
The backups for AWS clusters are encrypted using [AWS S3’s server-side encryption](https://docs.aws.amazon.com/AmazonS3/latest/dev/UsingServerSideEncryption.html) and the backups for GCP clusters are encrypted using [Google-managed server-side encryption keys](https://cloud.google.com/storage/docs/encryption/default-keys). 
```

File: src/current/cockroachcloud/frequently-asked-questions.md [hunk 2]
```diff
@@ -95,20 +95,20 @@ We do not automatically scale nodes based on your capacity usage. To add or remo
...
+Cockroach Labs runs full backups daily and incremental backups hourly for every {{ site.data.products.dedicated }} cluster. Full backups are retained for 30 days and incremental backups for 7 days. Only {{ site.data.products.dedicated }} cluster backups are available to users at this time. 
+
+Once a cluster is deleted, full backups are retained for 30 days and incremental backups for 7 days. If a user deletes the organization they will lose access to all of the backups that were taken from that cluster. See the [Use Managed-Service Backups](use-managed-service-backups.html?filters=dedicated#ways-to-restore-data) page for ways to restore data from your cluster's automatic backups in the Console.
 
 {{site.data.alerts.callout_info}}
 During [limited access](/docs/{{site.versions["stable"]}}/cockroachdb-feature-availability.html), managed backups are not available for {{ site.data.products.dedicated }} clusters on Azure. Customers can [take and restore from their own backups on Azure storage](take-and-restore-customer-owned-backups.html). Refer to [{{ site.data.products.dedicated }} on Azure](cockroachdb-dedicated-on-azure.html).
 {{site.data.alerts.end}}
```
Comment (MichaelTrestman) [inline src/current/cockroachcloud/frequently-asked-questions.md:105]: I suggest adding a "Cloud Provider considerations" subheader, and nesting the Azure, and AWS specific stuff (and any GCP stuff) under there.

**Review by MichaelTrestman — COMMENTED** [review]
Made a few suggestions to improve the helpfulness of this entry.

**Review by kathancox — COMMENTED** [review]
Just a couple notes, Michael's idea to rework this q a bit makes sense to me!

I'm not sure if you've checked in with Liv with this one, but perhaps cc'ing is worthwhile.

File: src/current/cockroachcloud/frequently-asked-questions.md [hunk 3]
```diff
@@ -95,20 +95,20 @@ We do not automatically scale nodes based on your capacity usage. To add or remo
 
 ### Who is responsible for backup?
 
-Cockroach Labs runs full backups daily and incremental backups hourly for every {{ site.data.products.dedicated }} cluster. Full backups are retained for 30 days and incremental backups for 7 days. Only {{ site.data.products.dedicated }} cluster backups are available to users at this time. See the [Use Managed-Service Backups](use-managed-service-backups.html?filters=dedicated#ways-to-restore-data) page for ways to restore data from your cluster's automatic backups in the Console.
+Cockroach Labs runs full backups daily and incremental backups hourly for every {{ site.data.products.dedicated }} cluster. Full backups are retained for 30 days and incremental backups for 7 days. Only {{ site.data.products.dedicated }} cluster backups are available to users at this time.
```
[the remark this answers is not in the document]
Comment (ashleykwtam) [reply src/current/cockroachcloud/frequently-asked-questions.md:98]: Yes good catch here - this should be removed as cluster backups are available to both

[the remark this answers is not in the document]
Comment (ashleykwtam) [reply src/current/cockroachcloud/frequently-asked-questions.md]: Thanks @MichaelTrestman for the suggestions! I really liked the intro on why customers should regularly take backups, as well as including a section for links to relevant pages.

**Comment by ashleykwtam** [comment]
Thanks @MichaelTrestman and @kathancox for the insightful suggestions to flesh out this section. I've kept the backup intro and reorganized the paragraphs so it flows more. I've put the Cloud provider info at the end of the section, though would love your thoughts on if it's better to move the Links section to follow it.

**Review by MichaelTrestman — APPROVED** [review]
LGTM!

**Pull request merged**